packages/front-end/components/Settings/EditDataSource/DataSourceMetrics.tsx

```diff
@@ -0,0 +1,275 @@
+                    datasource: dataSource.id,
+                    projects: dataSource.projects || [],
+                  },
+                  edit: false,
+                  duplicate: false,
+                })
+              }
+            >
+              <FaPlus className="mr-1" /> Add
+            </button>
+          ) : null}
+          <button
+            className="btn text-dark"
+            onClick={(e) => {
+              e.preventDefault();
+              setMetricsOpen(!metricsOpen);
+            }}
+          >
+            <FaChevronRight
+              style={{
+                transform: `rotate(${metricsOpen ? "90deg" : "0deg"})`,
+              }}
+            />
+          </button>
+        </div>
+      </div>
+      {metricsOpen ? (
+        <div className="my-3">
+          {metrics && metrics?.length > 0 ? (
+            <div>
+              {metrics.map((metric) => {
+                return (
+                  <div key={metric.id} className="card p-3 mb-3 bg-light">
+                    <Link href={`/metric/${metric.id}`}>
+                      <div
+                        className="d-flex flex-row align-items-center justify-content-between"
+                        role="button"
+                      >
+                        <div className="pr-3">
+                          <div className="mr-5 w-100">
+                            <h4
+                              className={
+                                metric.status === "archived" ? "text-muted" : ""
+                              }
+                            >
+                              {metric.name}
+                            </h4>
+                            <div className="d-flex flex-row align-items-center">
+                              <div className="pr-3">
+                                <strong
+                                  className={
+                                    metric.status === "archived"
+                                      ? "text-muted"
+                                      : ""
+                                  }
+                                >
+                                  Type:{" "}
+                                </strong>
+                                <code
+                                  className={
+                                    metric.status === "archived"
+                                      ? "text-muted"
+                                      : ""
+                                  }
+                                >
+                                  {metric.type}
+                                </code>
+                              </div>
+                              <div
+                                className={clsx(
+                                  metric.status === "archived"
+                                    ? "text-muted"
+                                    : "",
+                                  "pr-3"
+                                )}
```

Comment:
Just a tip on how clsx can accomplish this more concisely, looks like there are a few opportunities to do this in this file

```suggestion
                                className={clsx({
                                    "text-muted": metric.status === "archived"
                                  },
                                  "pr-3"
                                )}
```

Reply:
Addressed in this [commit](https://github.com/growthbook/growthbook/pull/1412/commits/f1affc08c34294aef3062e7f168a71af13fd0cce).